### ISSUE TITLE
bytes filter enhancement

### ISSUE THREAD

**Issue by dderjoel** [freearhey/vue2-filters#85]
I was wondering, why 'kb' is written in lowercase-letters whereas 'GB' is capital.

also, maybe one could pluralize and capitalize the byte-result

https://github.com/freearhey/vue2-filters/blob/master/src/other/bytes.js
```
return `${sizeInBytes} byte`; 
```
by
```
return sizeInBytes === 1 ? '1 Byte' : `${sizeInBytes} Bytes`;
```

**Comment by freearhey** [comment]
Honestly, I don't have any strict opinion on the matter. But it seems to me that if we change the notation of bytes, we should do the same with kilobytes.

We can also just change `byte` to `B`, and `kb` to `KB`.

**Comment by dderjoel** [comment]
Yes, I also think we should have that consistent.
I am fine with uppercase 'b' and all being abbreviations.
B
kB or KB, but I prefer KB although kilo (as in SI-Units) usually is lowercase.
MB
GB
TB
PB
EB
ZB
YB
XB

**Comment by freearhey** [comment]
Okay, then that's what I'll do.

**Comment by freearhey** [comment]
After more detailed study, I decided to follow your first sentence and replace "byte" with "bytes" in all cases except "1 byte". Kilobytes are now also referred to as "kB" to fully comply with SI Unit rules.

All this including `percent` filter is already available in version `v0.10.0`.
